torch/distributed/_tensor/ops/math_ops.py

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) Meta Platforms, Inc. and affiliates
-from typing import cast, List, Optional, Tuple
+from typing import cast, List, Optional, Sequence, Tuple
```

Comment:
In addition to the op level tests, I think we should add a test in `test_math_ops.py` to use `distribute_module` to partition a nn.LayerNorm, and mark input as sharded in `prepare_input_fn`, to make sure things work for the forward case when we apply this to nn.Module, this can be a follow up PR